playground/webpack.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = (env = {production: false}) => ({
     warningsFilter: /export .* was not found in/,
   },
   devServer: {
+    host: '0.0.0.0',
```

Comment:
```suggestion
```

You can just `yarn dev:host` https://github.com/Shopify/polaris-react#testing-on-mobile-or-a-virtual-machine